sentinelone/komand_sentinelone/actions/disable_agent/action.py

```diff
@@ -6,96 +6,240 @@
 
 class DisableAgent(insightconnect_plugin_runtime.Action):
     TIMEZONES_MAP = {
```

Comment:
You don't have to do it here, but next time you get in this code, make this map it's own file in the util's directory.

Reply:
Sure, will do. I usually run the black formatting at the end, and that's what changed the formatting on this action.